dev-ops/integration_test.user.env

```diff
@@ -9,10 +9,6 @@ APPLICATION_ENV = development
 # Database connectivity url
 MONGODB_URL = mongodb://mongo:27017/elevate-mentoring
```

Comment:
@priyanka-TL mongodb connection url not require

Reply:
@rakeshSgr Nevil is working on clearing mongo task so i skipped it